### PR TITLE
MLIBZ-2366: Fix Mobile Identity Connect on NativeScript

### PR DESCRIPTION
**Pull request by thomasconner**

### CODE REVIEW

File: src/nativescript/user/user.android.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { User as CoreUser } from '../../core/user';
+
+export class User extends CoreUser {
```
Comment (tejasranade) [inline src/nativescript/user/user.android.js]: Looks suspicious. Intentionally empty?

File: src/nativescript/user/user.ios.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { User as CoreUser } from '../../core/user';
+
+export class User extends CoreUser {
+  static handleMICRedirectURL(redirectUri, micRedirectURL) {
```
Comment (tejasranade) [inline src/nativescript/user/user.ios.js]: Who calls this method? I don't see a reference to this method name in the current master. I don't see any invocations in this PR either. What am I missing?

File: src/nativescript/user/user.android.js (hunk 1 above)
Comment (thomasconner) [reply src/nativescript/user/user.android.js]: This is no longer needed. I will remove it.

File: src/nativescript/user/user.ios.js (hunk 2 above)
Comment (thomasconner) [reply src/nativescript/user/user.ios.js]: This is no longer needed. I will remove it.

**Pull request merged**